.github/scripts/update-formula.sh

```diff
@@ -5,6 +5,14 @@ sha=$3
 if [ "$version" == '' ] || [ "$sha" == '' ]; then
   exit
 fi
+#fetch the current version
+current_version=$(grep -w "version " Formula/$formula.rb | awk '{print $2}' | tr -d \")
+echo "Current version $current_version"
+if [ "$version" != "$current_version" ]; then
+  cp Formula/$formula.rb Formula/$formula@$current_version.rb
+  class_name=$(echo $formula | sed -e "s/\b./\u&/g")AT$(echo $current_version | tr -dc '[:alnum:]\n\r' | sed 's/[[:alpha:]]/\U&/')
```

Comment:
Can you add images of this copy operation?

Reply:
Here is the image of the new formula created https://github.com/kridai/homebrew-brew/tree/main/Formula